projects/plugins/jetpack/tools/webpack.config.js

```diff
@@ -22,6 +22,15 @@ const sharedWebpackConfig = {
 	resolve: {
 		...baseWebpackConfig.resolve,
 		modules: [ path.resolve( path.dirname( __dirname ), '_inc/client' ), 'node_modules' ],
+
+		// Alias for @automattic/jetpack-in-place-connection.
+		alias: {
+			'@automattic/jetpack-in-place-connection': path.resolve(
+				path.dirname( path.dirname( path.dirname( __dirname ) ) ),
+				'rna',
+				'in-place-connection'
+			),
+		},
```

Comment:
Moving to a single monolithic `components` package like `@wordpress/components` or `@automattic/components` is something to strongly consider.

Each package requires a certain amount of infrastructure and maintenance (changelogs, build system, publishing, aliases)... all of that can be reduced to exactly 1 if you use a single components package.

Reply:
OTOH, the point of the Jetpack monorepo is to provide the infrastructure for changelogs, a build system, and even publishing so that individual projects don't have to concern themselves with it. We're still building that, but it's going well IMO.

For example, we currently have 28 different composer packages in the monorepo, that automatically get built and mirrored to standalone mirror repos that are then picked up by packagist. I'm working on the changelog handling now (currently none of our packages bother with changelogs 😒), and automatic release tagging is on my list too.

All a plugin has to do to pull them in from the monorepo is add a short snippet to their composer.json
```
	"repositories": [
		{
			"type": "path",
			"url": "../../packages/*",
			"options": {
				"monorepo": true
			}
		}
	],
```
I'm not familiar with the whole node ecosystem, but I'd start by looking at whether you can do something similar in package.json to add a "repository" that would let npm or yarn pull them in like any other dependency.

Reply:
Hi @sarayourfriend!
That's a good suggestion.
We've been discussing this in the P2, and so far we're leaning toward using larger packages (5 at the moment), with a possibility of adding `@automattic/jetpack-components` package for shared code between the packages.

Reply:
That's where the discussion happened: p9dueE-2sp-p2#comment-4392